torch_xla/core/xla_env_vars.py

```diff
@@ -20,3 +20,9 @@
 TPU_CHIPS_PER_HOST_BOUNDS = 'TPU_CHIPS_PER_HOST_BOUNDS'
 TPU_MESH_CTLER_ADDR = 'TPU_MESH_CONTROLLER_ADDRESS'
 TPU_MESH_CTLER_PORT = 'TPU_MESH_CONTROLLER_PORT'
+PJRT_DEVICE = 'PJRT_DEVICE'
```

Comment:
We should start adding comments describing the xla env vars -- maybe in a separate clean-up PR in the future.

Reply:
Other than `PJRT_DEVICE`, the rest of these topology variables apply to other TPU VM generally. I'll see if we can add a description to the Cloud TPU documentation and link it here.